src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class RightTriangle {
```

Comment:
The class `RightTriangle` is empty. If this is intended to be a model representing a right triangle, it should have properties such as base, height, and a method to calculate the area, adhering to the principles of object-oriented programming.